superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -126,12 +108,11 @@ class TableElement extends React.PureComponent {
       );
     }
     return header;
-  }
+  };
 
-  renderControls() {
+  const renderControls = () => {
     let keyLink;
-    const { table } = this.props;
-    if (table.indexes && table.indexes.length > 0) {
+    if (table?.indexes.length > 0) {
```

Comment:
if (table?.indexes?.length)

0 will be falsy, and it won't be negative. Also looks like indexes could be null or undefined.

Reply:
a lot of people still prefer foo.length > 0, so it's not wrong, just a personal preference. :)